spec/std/range_spec.cr

```diff
@@ -481,6 +481,18 @@ describe "Range" do
     it "works for other types" do
       ('a'..'c').size.should eq(3)
     end
+
+    it "raises on beginless range" do
+      expect_raises(ArgumentError, "Can't calculate size of an open range") do
+        ((true ? nil : 1)..3).size
```

Comment:
Why `nil..3` is not enough?

Reply:
Because #size raises at the compile-time, you need to have an union to invoke the runtime behaviour, other way of writing that would be `nil.as(Int32?)`.

Reply:
Ok. I missed that detail